docker/docker.go

```diff
@@ -359,3 +367,9 @@ func GetImageRegistryConfig(image string, prsMap map[string]v3.PrivateRegistry)
 	}
 	return "", "", nil
 }
+
+func convertToSemver(version string) (*semver.Version, error) {
+	compVersion := strings.SplitN(version, ".", 3)
+	compVersion[2] = "0"
```

Comment:
Lets add a check for compVersion len. In case someone unaware adds incorrect value to the map